pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql

```diff
@@ -0,0 +1,285 @@
+SELECT s.alias NAME
+        ,s.description description
+        ,s.NAME id
+        ,parent.NAME parentId
+        ,COALESCE(tt.sampleTypeName, sc.alias) sampleType
```

Comment:
How does this line work? My understanding is that it involves searching for TissueType.sampleTypeName column, but that doesn't exist in the database.

Reply:
`TissueType.sampleTypeName` does exist in OICR DB. Was added to preserve the old "sample type" in Pinery when we reduced the Tissue classes down to just "Tissue"

Reply:
Aha, I'd forgotten that was there.